references.bib

```diff
@@ -0,0 +1,278 @@
+  journal={Journal of game development},
+  volume={1},
+  number={1},
+  pages={7--28},
+  year={2004},
+  publisher={Citeseer}
+}
+%used
+@inproceedings{botea2011ultra,
+  title={Ultra-Fast Optimal Pathfinding without Runtime Search.},
+  author={Botea, Adi and others},
+  booktitle={AIIDE},
+  year={2011}
+}
+%used
+@inproceedings{cazenave2006optimizations,
+  title={Optimizations of data structures, heuristics and algorithms for path-finding on maps},
+  author={Cazenave, Tristan},
+  booktitle={Computational Intelligence and Games, 2006 IEEE Symposium on},
+  pages={27--33},
+  year={2006},
+  organization={IEEE}
+}
+%used
+@article{cui2011based,
+  title={A*-based pathfinding in modern computer games},
+  author={Cui, Xiao and Shi, Hao},
+  journal={International Journal of Computer Science and Network Security},
+  volume={11},
+  number={1},
+  pages={125--130},
+  year={2011},
+  publisher={International Journal of Computer Science and Network Security (IJCSNS)}
+}
+%used
+@article{cui2011direction,
+  title={Direction oriented pathfinding in video games},
+  author={Cui, Xiao and Shi, Hao},
+  journal={International Journal of Artificial Intelligence \& Applications},
+  volume={2},
+  number={4},
+  pages={1},
+  year={2011},
+  publisher={Academy \& Industry Research Collaboration Center (AIRCC)}
+}
+%used
+@article{algfoor2015comprehensive,
+  title={A comprehensive study on pathfinding techniques for robotics and video games},
+  author={Algfoor, Zeyad Abd and Sunar, Mohd Shahrizal and Kolivand, Hoshang},
+  journal={International Journal of Computer Games Technology},
+  volume={2015},
+  pages={7},
+  year={2015},
+  publisher={Hindawi Publishing Corp.}
+}
+%used
+@inproceedings{kallmann2010navigation,
+  title={Navigation queries from triangular meshes},
+  author={Kallmann, Marcelo},
+  booktitle={International Conference on Motion in Games},
+  pages={230--241},
+  year={2010},
+  organization={Springer}
+}
+%used
+@article{korf1990real,
+  title={Real-time heuristic search},
+  author={Korf, Richard E},
+  journal={Artificial intelligence},
+  volume={42},
+  number={2-3},
+  pages={189--211},
+  year={1990},
+  publisher={Elsevier}
+}
+%used
+@article{sturtevant2007memory,
+  title={Memory-Efficient Abstractions for Pathfinding.},
+  author={Sturtevant, Nathan R},
+  journal={AIIDE},
+  volume={684},
+  pages={31--36},
+  year={2007}
+}
+%used
+@article{lawrence2013database,
+  title={Database-driven real-time heuristic search in video-game pathfinding},
+  author={Lawrence, Ramon and Bulitko, Vadim},
+  journal={IEEE Transactions on Computational Intelligence and AI in Games},
+  volume={5},
+  number={3},
+  pages={227--241},
+  year={2013},
+  publisher={IEEE}
+}
+%used
+@inproceedings{harabor2010breaking,
+  title={Breaking Path Symmetries on 4-Connected Grid Maps.},
+  author={Harabor, Daniel and Botea, Adi and others},
+  booktitle={AIIDE},
+  year={2010}
+}
+%used
+@article{lester2005pathfinding,
```

Comment:
Formatting issues with this one. Should generally use the `@misc` type for websites, and wrap the URL in `\url{}` for proper formatting. Also write "accessed" in English, not whatever language "Acesso em" is ;)